### PR TITLE
Fix issue #1805

### PR DESCRIPTION
**Pull request by Minilfat**
That issue is actual not only for iOS devices but Android browser as well. I wrote more detailed [here](https://github.com/Modernizr/Modernizr/issues/1805#issuecomment-495656198). 
Idea how to fix this issue was suggested [here](https://github.com/Modernizr/Modernizr/issues/1805#issuecomment-284103672).

### CODE REVIEW

**Comment by rejas** [comment]
Thx for the PR but I got a quetion: This basically reverts https://github.com/Modernizr/Modernizr/pull/2264 which tried to fix https://github.com/Modernizr/Modernizr/issues/1569 
SO my question is: how can we get a solution that dixes both problems?

**Comment by Minilfat** [comment]
> Thx for the PR but I got a quetion: This basically reverts #2264 which tried to fix #1569
> SO my question is: how can we get a solution that dixes both problems?

Ohh, I was not aware about those issues. I will let you know if we come up with some solution. Thank you

**Comment by Minilfat** [comment]
> Thx for the PR but I got a quetion: This basically reverts #2264 which tried to fix #1569
> SO my question is: how can we get a solution that dixes both problems?

@rejas  hello. What do you think about this solution: we set the height of the element to 50vh, but also set "max-height" to be 10px. In case the "vh" is supported by browser then the height of the element is 10px.

The code is following: 
```javascript
define(["Modernizr", "testStyles", "computedStyle"], function(Modernizr, testStyles, computedStyle) {
  testStyles("#modernizr { height: 50vh; max-height: 10px; }", function(elem) {
    var compStyle = parseInt(computedStyle(elem, null, "height"), 10);
    Modernizr.addTest("cssvhunit", compStyle === 10);
  });
});
```

File: feature-detects/css/vhunit.js [hunk 1]
```diff
@@ -14,11 +14,9 @@
...
 !*/
-define(['Modernizr', 'testStyles', 'computedStyle', 'roundedEquals'], function(Modernizr, testStyles, computedStyle, roundedEquals) {
-  testStyles('#modernizr { height: 50vh; }', function(elem) {
-    var height = parseInt(window.innerHeight / 2, 10);
+define(['Modernizr', 'testStyles', 'computedStyle'], function(Modernizr, testStyles, computedStyle) {
+  testStyles('#modernizr { height: 50vh; max-height: 10px; }', function(elem) {
     var compStyle = parseInt(computedStyle(elem, null, 'height'), 10);
-
-    Modernizr.addTest('cssvhunit', roundedEquals(compStyle, height));
+    Modernizr.addTest('cssvhunit', compStyle == 10);
```
Comment (rejas) [inline feature-detects/css/vhunit.js]: why == instead of === ?

Comment (Minilfat) [reply feature-detects/css/vhunit.js]: Both of them have type "Number", don't they? I can change to '===' not a problem at all. Should I do it?

Comment (rejas) [reply feature-detects/css/vhunit.js]: === is in 99 % of the time preferable even if you think that the types are always the same...

Comment (Minilfat) [reply feature-detects/css/vhunit.js]: Fixed! :)

**Pull request merged**